blob/gcsblob/example_test.go

```diff
@@ -0,0 +1,48 @@
+// Copyright 2018 The Go Cloud Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package gcsblob_test
+
+import (
+	"context"
+
+	"github.com/google/go-cloud/blob/gcsblob"
+	"github.com/google/go-cloud/gcp"
+)
+
+func ExampleOpenBucket() {
+	ctx := context.Background()
+
+	// Load credentials for GCP.
+	// This example uses the default approach; see
+	// https://cloud.google.com/docs/authentication/production
+	// for more info on alternatives.
+	creds, err := gcp.DefaultCredentials(ctx)
+	if err != nil {
+		return
+	}
+
+	// Create an HTTP client.
+	// This example uses the default HTTP transport and the credentials created
+	// above.
+	client, err := gcp.NewHTTPClient(gcp.DefaultTransport(), gcp.CredentialsTokenSource(creds))
+	if err != nil {
+		return
+	}
+
+	// Create a *blob.Bucket.
+	_, _ = gcsblob.OpenBucket(ctx, "my-bucket", client, nil)
+
+	// Output:
```

Comment:
was there intended to be more below this?

Reply:
No. In "example" tests, you put the expected output under a `// Output` comment like this, and the test "passes" if the actual output matches the expected.

https://blog.golang.org/examples

This test doesn't output anything, so there's nothing under the comment.

Depending on the environment, `OpenBucket` here may or may not work, so it's hard to output anything interesting.